### ISSUE TITLE
Outdated MCO in 4.3 playload

### ISSUE THREAD

**Issue by vrutkovs** [openshift/okd#42]
MCO from `fcos` branch is being promoted to 4.4 release, but not in 4.3.

`fcos` branch has been rebased recently on latest master, so its safe to point `4.3:machine-config-operator` to 4.4 image. This would unblock 4.3 installs.

/assign @smarterclayton

**Comment by punkprzemo** [comment]
Hi,
will that fix the below error for MCO in OKD 4.3 ?
```
oc describe co machine-config | grep Message
    Message:               Cluster not available for 4.3.0-0.okd-2020-01-29-124051
    Message:               Cluster is bootstrapping 4.3.0-0.okd-2020-01-29-124051
    Message:               Failed to resync 4.3.0-0.okd-2020-01-29-124051 because: timed out waiting for the condition during syncRequiredMachineConfigPools: pool master has not progressed to latest configuration: configuration status for pool master is empty: pool is degraded because nodes fail with "3 nodes are reporting degraded status on sync": "Node node057.okd4.example.com is reporting: \"unexpected on-disk state validating against rendered-master-eae5c7efcd16a07ad1927802ff30efe5\", Node node059.okd4.example.com is reporting: \"unexpected on-disk state validating against rendered-master-eae5c7efcd16a07ad1927802ff30efe5\", Node node058.okd4.example.com is reporting: \"unexpected on-disk state validating against rendered-master-eae5c7efcd16a07ad1927802ff30efe5\"", retrying
```
If yes, when it's going to be promoted in 4.3? Is there workaround for OKD 4.3?
Currently with fresh 4.3 bare metal install only MCO is in degraded state. rest seems to be OK.

**Comment by gtema** [comment]
> Hi,
> will that fix the below error for MCO in OKD 4.3 ?

I guess no, the cause for issue you mention (which I also have) lies in https://github.com/openshift/machine-config-operator/issues/1378

**Comment by parthakom2** [comment]
Hi,

Any idea when this fix will be available or a workaround for OKD 4.3. Its blocking any work related to 4.3.

**Comment by vrutkovs** [comment]
OKD 4.3 is not longer an active stream
